### PR TITLE
add Ambermoon.net, a remake of Ambermoon

### PR DESCRIPTION
**Pull request by kparal**
Please note, while I used `content: free`, because the content is bundled in the release archive and free for the user, I believe the author actually uses the original assets from Ambermoon, i.e. most probably copyrighted content. I assume they are relying on its abandonware status and take the liberty to distribute it. I'm not sure if the `content` field should be changed to `commercial` or not.

### CODE REVIEW

File: games/a.yaml [hunk 1]
```diff
@@ -255,6 +255,28 @@
...
+  originals:
+    - Ambermoon
+  repo: https://github.com/Pyrdacor/Ambermoon.net
+  development: very active
+  status: playable
+  lang:
+    - 'C#'
+  license:
+    - GPL3
+  content: free
```
Comment (cxong) [inline games/a.yaml:269]:
```suggestion
  content: commercial
```
since abandonware isn't a license

File: games/a.yaml [hunk 2]
```diff
@@ -255,6 +255,28 @@
...
 
+- name: Ambermoon.net
+  type: remake
+  originals:
+    - Ambermoon
+  repo: https://github.com/Pyrdacor/Ambermoon.net
+  development: very active
+  status: playable
+  lang:
+    - 'C#'
```
Comment (cxong) [inline games/a.yaml:266]:
```suggestion
    - C#
    framework:
    - .NET
```

**Review by cxong — CHANGES_REQUESTED** [review]

**Comment by kparal** [comment]
Updated per review.

**Review by cxong — APPROVED** [review]

**Comment by kparal** [comment]
What is the next step now?

**Pull request merged**